README.rst

```diff
@@ -46,6 +46,14 @@ Current features
 * Search over existing studies (see known issues).
 * Generate basic visualizations with the available studies and datasets.
 
+Accepted raw files
+------------------
+
+* Multiplexed SFF
+* Multiplexed FASTQ: forward, reverse (optional), and barcodes
+* Per sample FASTQ: forward
+* Multiplexed fasta/qual files.
```

Comment:
fasta -> FASTA and perhaps remove the period at the end of this sentence or add it to the other points? ... that is of course a minor suggestion, but yeah I noticed that :neckbeard: